bird.js

```diff
@@ -12,7 +12,7 @@ class Bird {
     this.x = 64;
 
     this.gravity = 0.6;
-    this.lift = -15;
+    this.delay = 0;
```

Comment:
Would a `boolean` flag variable make more sense here?

Reply:
Yes, you are right. A boolean variable would work, and it's going ot be easier to read. Can I correct it by making another pull request?